src/main/java/io/passport/server/controller/PassportController.java

```diff
@@ -0,0 +1,107 @@
+package io.passport.server.controller;
+
+import io.passport.server.model.Passport;
+import io.passport.server.service.PassportService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Class which stores the generated HTTP requests related to passport operations.
+ */
+@RestController
+@RequestMapping("/passport")
+public class PassportController {
+
+
+    private static final Logger log = LoggerFactory.getLogger(PassportController.class);
+    /**
+     * Passport service for passport management
+     */
+    private final PassportService passportService;
+
+    @Autowired
+    public PassportController(PassportService passportService) {
+        this.passportService = passportService;
+    }
+
+
+    /**
+     * Read all passports
+     * @return
+     */
+    @GetMapping()
+    public ResponseEntity<List<Passport>> getAllPassports() {
+        List<Passport> passports = this.passportService.getAllPassports();
+
+        long totalCount = passports.size();
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.add("X-Total-Count", String.valueOf(totalCount));
+
+        return ResponseEntity.ok().headers(headers).body(passports);
+    }
+
+
```

Comment:
Newline left from deleted content. You can remove them along with the other marked newlines.